docs/blog/2023-conference-announcements.rst

```diff
@@ -0,0 +1,109 @@
+.. post:: September 26, 2022
+   :tags: portland-2023, prague, atlantic-2023, newsletter
+
+Announcing Write the Docs conferences for 2023
+==============================================
```

Comment:
Can we dupe this to both conf news folders so that it gets conference formatting? Or will that create more problems than it will resolve?

Reply:
I think it will create problems but we can try, but more, I think it will actually be more confusing to have the same text on the website in multiple URLs with multiple designs.

Reply:
Yeah, it's just the first thing you can read on the new conf sites, and it looks kinda rubbish when it falls back to the default look.